### ISSUE TITLE
smartling.yml file with id, secret account, being told no secret

### ISSUE THREAD

**Issue by enegron-sl** [Smartling/smartling-cli#59]
I have a smartling.yml file with:

user_id:
    "user_id"
secret:
    "secret"

account_id:
    "account_id"

I run projects list and get error:

smartling-cli projects list
ERROR: Cannot find mandatory configuration parameter "token secret"

Please, specify either:
- Environment variable $SMARTLING_SECRET;
- Command line option --secret=<secret>;
- Or set "secret" option in the configuration file:

	/Users/eric/cli-tests/MLC/smartling.yml
		secret: "PUT_VALUE_HERE"

**Comment by dimitrystd** [comment]
Is it about wording? or i didn't get task

**Comment by enegron-sl** [comment]
No it’s about functionality. If my config yaml file has user id, secret, and account ID then projects list should work. I feel like I reported something similar before?

**Comment by dericko** [comment]
Any updates on this? I'm getting the same issue as @enegron-sl, where my `smartling.yml` is correctly configured by `smartling-cli init`, but the cli cannot find it:

```
derickolson@dlo-mbp ~/projects $ smartling-cli projects list
ERROR: Cannot find mandatory configuration parameter "token secret"

Please, specify either:
- Environment variable $SMARTLING_SECRET;
- Command line option --secret=<secret>;
- Or set "secret" option in the configuration file:

	/Users/derickolson/projects/smartling.yml
		secret: "PUT_VALUE_HERE"
```
Also isn't detecting it when I set the env variable `$SMARTLING_SECRET`

Temporarily getting around this by passing my secret into the command each time `smartling-cli projects list --secret={my_secret}`

**Comment by shaneog** [comment]
Any update on this? I also have this issue.

**Comment by dimitrystd** [comment]
Fixed in v1.4
